S23/thofstrand/lowerCase.py

```diff
@@ -0,0 +1,91 @@
+import pandas as pd
+
+
+
+def lowerCase(word, lang):
+    word = word.strip()
+    lang = lang.split('-')
+    langStart = lang[0]
+
+    if len(langStart) != 2:
```

Comment:
You should look at the BCP-47 spec.  There are (many) language codes that have length 3.